README.md

```diff
@@ -36,6 +36,14 @@ UPPHUB_BEGIN
       "category": "scientific",
       "readme": "https://raw.githubusercontent.com/mirek-fidler/gdal/master/README.md"
     }
+    { "name": "MessageCtrl",
```

Comment:
You forget to add comma before object.

Reply:
@klugier 
OK, I have fixed it. I am having hard times with my keyboard lately...